artist-traits-rx/src/main/kotlin/com/uber/artist/traits/rx/KotlinApiHelper.kt

```diff
@@ -165,7 +163,6 @@ fun addRxBindingApiForSettable(type: TypeSpec.Builder, api: KotlinSettableApi, i
             .addKdoc(rxBindingMethodDoc)
             .apply {
                 if (api.isUViewOverride) {
-                    addAnnotation(Override::class.java)
```

Comment:
why is this removed? will kotlinpoet/kotlin add the Override annotation if you add that modifier?

Reply:
kotlin uses the `override` modifier, the annotation is redundant/not idiomatic